android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -102,6 +103,17 @@ Activity getCurrentActivity() {
 
     public void initNativeSdk(final ReadableMap rnOptions, Promise promise) {
         SentryAndroid.init(this.getReactApplicationContext(), options -> {
+            if (rnOptions.hasKey("_metadata")) {
+                @Nullable final ReadableMap metadata = rnOptions.getMap("_metadata");
+                @Nullable final ReadableMap sdkMap = metadata != null ? metadata.getMap("sdk") : null;
+                @Nullable final String sdkMapName = sdkMap != null ? sdkMap.getString("name") : null;
+                @Nullable final String sdkMapVersion = sdkMap != null ? sdkMap.getString("version") : null;
+                if (sdkMapName != null && sdkMapVersion != null) {
+                    @NonNull final SdkVersion sdkVersion = new SdkVersion(sdkMapName, sdkMapVersion);
+                    options.setSentryClientName(sdkMapName + "/" + sdkMapVersion);
+                    options.setSdkVersion(sdkVersion);
```

Comment:
We need to be sure that this is working, I might have not tested well.
See https://github.com/getsentry/team-mobile/issues/48#issuecomment-1424684160

Reply:
I've only checked that the set SDK name appears in the `event.sdk` in Sentry.

Reply:
M: There's a simpler approach here https://github.com/getsentry/sentry-dart/pull/1320/files
Since we just need to append `.flutter` at the end of the name.